lib/kubernetes-deploy/logger.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'logger'
```

Comment:
I agree with Katrina's point. It's quite sketchy that requiring `kubernetes-deploy/logger.rb` extends `KubernetesDeploy` module when the file is supposed to define `KubernetesDeploy::Logger`.

Why can't we put it into `KubernetesDeploy::Logger` and include it into `KubernetesDeploy`?